source/features/conversation-activity-filter.tsx

```diff
@@ -197,8 +196,8 @@ async function addWidget(header: string, state: State): Promise<void> {
 }
 
 const minorFixesIssuePages = [
-	getRghIssueUrl(5222),
-	getRghIssueUrl(4008),
+	'https://github.com/refined-github/refined-github/issues/5222',
```

Comment:
It is possible to make this a little nicer, unfortunately still not clickable😅

<img width="663" alt="image" src="https://user-images.githubusercontent.com/44045911/179344235-4ec42a64-0522-47c0-a9e5-459e23fd6358.png">